spec/core/urlParsing.ts

```diff
@@ -45,7 +45,7 @@ describe("urlParsing.ts", () => {
 	/* tslint:disable: no-string-literal */
 	describe("parsePath", () => {
 		for (const path in testCases) {
-			if (testCases.hasOwnProperty(path)) {
+			if (Object.prototype.hasOwnProperty.call(testCases, path)) {
```

Comment:
this looks javascript like.  which rule required this?

Reply:
The rule that requires this - no-prototype-builtins
[](https://stackoverflow.com/questions/39282873/how-do-i-access-the-object-prototype-method-in-the-following-logic)
[](https://stackoverflow.com/questions/12017693/why-use-object-prototype-hasownproperty-callmyobj-prop-instead-of-myobj-hasow)